.github/workflows/pr-title-lint.yml

```diff
@@ -0,0 +1,87 @@
+name: Lint Pull Request Title
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+
+permissions:
+  contents: read
+
+jobs:
+  lint_commit:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Commitlint
+        shell: bash
+        run: npm install --location=global @commitlint/cli@17.1.1
```

Comment:
can we maybe use https://github.com/amannn/action-semantic-pull-request should help avoid the extra configuration file?